index.html

```diff
@@ -154,7 +154,7 @@ <h3 class="framework-text">Skills</h3>
             <p class="text-form">
                 I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
             </p>
-            <form>
+            <form action="https://formspree.io/f/mayzozrb" method="POST">
                 <label for = "name"></label>
                 <input type="text" id="name" class="input-padron" required placeholder="Full name">
```

Comment:
- [OPTIONAL] I will highly recommend you to add this `name="name"` inside this input to make the form submission work properly.
- [OPTIONAL] Also consider to add this `name="email"` in the line `163` inside input.